GCTC-NTGC/gc-digital-talent
# Feature - Add product_id, email_config_id parameters to support API payload

Resolves #4294.

## Steps to test

1. Create a test Freshdesk account (if you don't have one already)
2. Add environment variables (`FRESHDESK_API_PRODUCT_ID`, `FRESHDESK_API_EMAIL_CONFIG_ID`) to api/.env based on api/.env.example from test Freshdesk account
3. Clear config cache `php artisan config:cache` (inside shell)
4. Verify successful ticket creation with product_id, email_config_id parameters set
